README.rst

```diff
@@ -17,30 +17,97 @@
 
 .. index-start-marker
 
-Installer for D-Wave's Ocean Tools.
+`Ocean <https://docs.ocean.dwavesys.com/en/stable>`_ is
+`D-Wave's <https://www.dwavesys.com>`_ suite of tools for solving hard problems
+with quantum computers.
 
+.. index-end-marker
 
 Installation
-------------
+============
 
 .. installation-start-marker
 
-Compatible with Python 3.6+:
+Installation from `PyPI <https://pypi.org/project/dwave-ocean-sdk/>`_:
 
 .. code-block:: bash
 
     pip install dwave-ocean-sdk
 
-To install from source:
+For more information, see the Ocean documentation's
+`installation <https://docs.ocean.dwavesys.com/en/stable/overview/install.html>`_
+page.
 
-.. code-block:: bash
+.. installation-end-marker
 
-    python setup.py install
+Getting Started
+===============
 
-.. installation-end-marker
+Sign up for the Leap quantum cloud service here:
+`Leap signup <https://cloud.dwavesys.com/leap/signup>`_. Leap provides a
+browser-based integrated development environment (IDE) with Ocean preinstalled.
+
+Start learning with the following D-Wave resources:
+
+* `System Documentation <https://docs.dwavesys.com/docs/latest/index.html>`_ to
+  learn about quantum computers and how to use them.
+
+* `Getting Started with Ocean <https://docs.ocean.dwavesys.com/en/stable/getting_started>`_
+  to install and start coding with Ocean software.
+
+* `dwave-examples <https://github.com/dwave-examples>`_ for code examples
+  and Jupyter Notebooks.
+
+* `D-Wave website <https://www.dwavesys.com/learn/resource-library>`_ for
+  whitepapers and additional resources.
+
+Example Quantum Program
+-----------------------
+
+The following lines of code solve and visualize a
+`random <https://docs.ocean.dwavesys.com/en/stable/docs_dimod/reference/generators.html>`_
+problem on a quantum computer.
```

Comment:
I understand it's easier to use a random problem for this short example, but can we also add a sentence that points to some real-world / application problems. Perhaps your getting started examples in the docs?

Reply:
Added
![image](https://user-images.githubusercontent.com/34041130/144870851-4fbf72b9-0448-4243-afd0-7b344a55cb19.png)

Reply:
Feels a bit repetitive of the bullet list above?

Reply:
Yes, but we also wanted to follow the random example with "real-world / application problems ... your getting started examples". I think users are less annoyed by a single repetitive sentence at the end of a chunk of text than frustrated by not finding breadcrumbs to the next place they are likely to want to visit

Reply:
Sounds good.